charts/fleet/values.yaml

```diff
@@ -76,6 +76,14 @@ tolerations: []
 ## Pod affinity for the controllers.
 affinity: {}
 
+resources: {}
```

Comment:
Nice, should we perhaps clarify that these resource constraints would apply to each deployment separately (fleet-controller, gitOps, helmOps controllers) and not be propagated to agents? That last bit may be obvious to most, but just to be on the safe side as the controller also manages agents.